src/Adapters/Storage/Mongo/MongoStorageAdapter.js

```diff
@@ -565,34 +565,7 @@ export class MongoStorageAdapter implements StorageAdapter {
         }
       }
       if (stage.$match) {
-        for (const field in stage.$match) {
-          if (schema.fields[field] && schema.fields[field].type === 'Pointer') {
-            const transformMatch = { [`_p_${field}`] : `${schema.fields[field].targetClass}$${stage.$match[field]}` };
-            stage.$match = transformMatch;
-          }
-          else if (schema.fields[field] && schema.fields[field].type === 'Date') {
-            const transformMatch = { [`${field}`]: new Date(stage.$match[field]) };
-            stage.$match = transformMatch;
-          }
-          if (field === 'objectId') {
-            const transformMatch = Object.assign({}, stage.$match);
-            transformMatch._id = stage.$match[field];
-            delete transformMatch.objectId;
-            stage.$match = transformMatch;
-          }
-          else if (field === 'createdAt') {
-            const transformMatch = Object.assign({}, stage.$match);
-            transformMatch._created_at = stage.$match[field];
-            delete transformMatch.createdAt;
-            stage.$match = transformMatch;
-          }
-          else if (field === 'updatedAt') {
-            const transformMatch = Object.assign({}, stage.$match);
-            transformMatch._updated_at = stage.$match[field];
-            delete transformMatch.updatedAt;
-            stage.$match = transformMatch;
-          }
-        }
+        this._parseAggregateArgs(schema, stage.$match);
```

Comment:
Returned value seem unused.

Reply:
Thanks for picking up on that...I was trying to hurry and pushed the repo by accident (muscle memory from my day job, haha). I just pushed the fix.